h2o-core/src/main/java/water/rapids/Env.java

```diff
@@ -288,6 +290,9 @@ static void init(AstPrimitive ast, String name) {
     init(new AstSeq());
     init(new AstSeqLen());
 
+    // Custom (eg. algo-specific)
+    for (AstPrimitive prim : PrimsService.INSTANCE.getAllPrims())
+      init(prim);
```

Comment:
Nice, thanks Michal for that! btw: later i will need to document all extension points we are having  (tracked here; https://0xdata.atlassian.net/browse/PUBDEV-4272)